.envrc

```diff
@@ -0,0 +1,3 @@
+if has nix; then
+  use nix
+fi
```

Comment:
Missing newline?

Reply:
It doesn't matter for processing the env file but I'll add it if I have to push to the PR again. Still working on figuring out why CI is failing when tests are fine locally.